src/NeoSharp.Core/Converters/IPEndPointConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using NeoSharp.BinarySerialization;
+using NeoSharp.BinarySerialization.SerializationHooks;
+using NeoSharp.Core.Extensions;
+
+namespace NeoSharp.Core.Converters
+{
+    class IPEndPointConverter : IBinaryCustomSerializable
```

Comment:
Where do we use it over EndPointConverter?

Reply:
Is a different serialization, because we can't serialize the type